packages/highlight-vdom/src/Highlight.tsx

```diff
@@ -78,17 +95,19 @@ export function createHighlightComponent({
     Fragment,
   });
 
-  return function Highlight({
-    parts,
-    highlightedTagName = 'mark',
-    nonHighlightedTagName = 'span',
-    separator = ', ',
-    className,
-    classNames = {},
-    ...props
-  }: HighlightProps) {
+  return function Highlight(props: HighlightProps) {
```

Comment:
Why refactor this?

Reply:
ah I guess it's no longer needed now, but previously `className` wasn't in the allowed props